<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelloWorldDlg</class>
 <widget class="QDialog" name="HelloWorldDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Automation Group</string>
  </property>
  <widget class="QPushButton" name="but_Auto">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>530</y>
     <width>65</width>
     <height>65</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QMdiArea" name="mdiArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>520</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="but_Manual">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>530</y>
     <width>65</width>
     <height>65</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="but_Teach">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>530</y>
     <width>65</width>
     <height>65</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="but_Data">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>530</y>
     <width>65</width>
     <height>65</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
